spec/genre_spec.rb

```diff
@@ -9,12 +9,12 @@
       expect(genre).to have_attributes(name: 'Abc')
       expect(genre).to have_attributes(items: [])
     end
-    it 'add an item to a genre (add_item method)', positive: true do
-      genre = Genre.new name: 'Abc'
+    # it 'add an item to a genre (add_item method)', positive: true do
+    #   genre = Genre.new name: 'Abc'
```

Comment:
[OPTIONAL]: Please remove codes you are not applying to have a clean code. You can always find it from the GitHub version controlling 😊